frontend/tests/playwright/troubleshooting/loading.spec.ts

```diff
@@ -58,11 +58,16 @@ test.describe('Vulcan Page Content and SEO', () => {
          },
       ];
 
-      langsToHrefs.forEach(async (langToHref) => {
+      const checkLang = async function (langToHref: {
+         lang: string;
+         href: string;
+      }): Promise<void> {
          const hrefLang = page.locator(`link[hreflang="${langToHref.lang}"]`);
          await expect(hrefLang).toHaveAttribute('hreflang', langToHref.lang);
          await expect(hrefLang).toHaveAttribute('href', langToHref.href);
```

Comment:
```suggestion
         return Promise.all([
            expect(hrefLang).toHaveAttribute('hreflang', langToHref.lang),
            expect(hrefLang).toHaveAttribute('href', langToHref.href)
         ]);
```

Might as well run those in parallel too.